site/documentation/content/api/tenant/index.md

```diff
@@ -177,6 +177,7 @@ The table below provides an overview of the members of a JSON object representin
 | *subject-dn*             | *yes*      | *string*      | `-`           | The subject DN of the trusted root certificate in the format defined by [RFC 2253](https://www.ietf.org/rfc/rfc2253.txt). |
 | *public-key*             | *yes*      | *string*      | `-`           | The Base64 encoded binary DER encoding of the trusted root certificate's public key. |
 | *algorithm*              | *no*       | *string*      | `RSA`         | The name of the public key algorithm. Supported values are `RSA` and `EC`. |
+| *auto-provisioning-enabled* | *no*    | *boolean*     | `false`       | If set to `true`, protocol adapters MAY try to request auto-provisioning of devices that authenticate with a client certificate issued by this CA. Otherwise, protocol adapters MUST NOT request auto-provisioning. |
```

Comment:
FMPOV a protocol adapter should assume that the registry supports auto-provisioning if this property is included and set to `true`, i.e. the adapter then MAY request the registry to automatically provision the device based on its client certificate (and use the status code to determine the outcome). My point is that an adapter MAY not only *try* to do this but it should assume that the registry actually supports doing so ...